.codeclimate.yml

```diff
@@ -7,6 +7,11 @@ engines:
       - javascript
   eslint:
     enabled: true
+    checks:
```

Comment:
CodeClimate is using an older version of eslint, hence these false positives.

See: https://github.com/codeclimate/codeclimate-eslint/issues/76